server/sonar-web/src/main/js/components/SelectList/SelectListListContainer.tsx

```diff
@@ -23,6 +23,7 @@ import SelectListListElement from './SelectListListElement';
 
 interface Props {
   elements: string[];
+  disabledElements: string[];
```

Comment:
Minor: I'd make `disabledElements` optional here, it'd avoid the redundant line `disabledElements={Array []}` in snapshots.